src/ui/public/template_vis_type/template_renderbot.js

```diff
@@ -1,25 +1,27 @@
 import _ from 'lodash';
-import VisRenderbotProvider from 'ui/vis/renderbot';
-export default function TemplateRenderbotFactory(Private, $compile, $rootScope) {
-  var Renderbot = Private(VisRenderbotProvider);
+define(function (require) {
```

Comment:
You're reintroducing unnecessary AMD stuff here.  This file should continue to use es6 modules.